plugin/trino-hive/src/main/java/io/trino/plugin/hive/procedure/SyncPartitionMetadataProcedure.java

```diff
@@ -172,6 +172,9 @@ private static Set<String> doListPartitions(TrinoFileSystem fileSystem, Location
             if (!isValidPartitionPath(path, column, caseSensitive)) {
                 continue;
             }
+            if (!caseSensitive) {
```

Comment:
Please follow the commit message guideline: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages